build-tools-internal/version.properties

```diff
@@ -1,8 +1,8 @@
 elasticsearch     = 8.0.0
 lucene            = 8.9.0
 
-bundled_jdk_vendor = adoptopenjdk
-bundled_jdk = 16.0.1+9
+bundled_jdk_vendor = openjdk
```

Comment:
Does this impact the HOMER version check stuff?

Reply:
It would. We might want to disable the checks temporarily.